modules/transactions.js

```diff
@@ -660,13 +673,8 @@ Transactions.prototype.shared = {
 					return setImmediate(cb, 'Transaction not found');
 				}
 
-				if (transaction.type === 3) {
-					__private.getVotesById(transaction, function (err, transaction) {
```

Comment:
@SargeKhan It's OK to remove it because we can break backward compatibility in `1.0.0`